### ISSUE TITLE
Question about results of "Questions" dataset

### ISSUE THREAD

**Issue by Rupam-kumawat** [LUOyk1999/tunedGNN#12]
I am currently working to reproduce the results of your GNNs (GCN, SAGE, and GAT) on the "Questions" dataset, as presented in your paper. I am using the official code provided in your public repository for this evaluation. While configuring the experiment, I noticed a discrepancy in the specified learning rate between the paper and the accompanying bash file. I have attempted to run the model using both values but have thus far been unable to replicate the performance metrics reported in your publication. To ensure my evaluation is configured correctly, could you please provide clarification on the exact set of hyperparameters used to generate the final results for the "Questions" dataset? Any guidance you could offer would be greatly appreciated.

**Comment by LUOyk1999** [comment]
Thanks for your interest in our work and for taking the time to reproduce the results. Could you share the exact numbers you’re getting? I’m happy to take a closer look.

**Comment by Rupam-kumawat** [comment]
Thank you for your prompt response.
When I use a learning rate of 3e-5 on GCN and Sage, I obtain AUC-ROC scores of 0.7478 and 0.7024, respectively.
When I use a learning rate of 0.001 on GCN and Sage, I obtain AUC-ROC scores of 0.6485 and 0.7179, respectively.

python main.py --gnn gcn --dataset questions --pre_linear --hidden_channels 512 --epochs 1500 --lr 3e-5 --runs 3 --local_layers 10 --weight_decay 0.0 --dropout 0.3 --metric rocauc --device $1 --res
python main.py --gnn sage --dataset questions --pre_linear --hidden_channels 512 --epochs 1500 --lr 3e-5 --runs 3 --local_layers 6 --weight_decay 0.0 --dropout 0.2 --metric rocauc --device $1 --ln                 

These are the parameters I am using as specified in the given bash file.

**Comment by LUOyk1999** [comment]
Ok, the parameters look fine. Could you please let me know which versions of GPU, Python, PyTorch, and PyG you are using? Also, could you share the source of the dataset and how the split was generated?

**Comment by Rupam-kumawat** [comment]
I am getting the dataset from the HeterophilousGraphDataset class in torch_geometric, and the split is generated using the load_fixed_splits function in your data_utils.py file. 
The specific versions are: Python:3.11.7, PyTorch: 2.5.1+cu121, PyG: 2.6.1.

**Comment by LUOyk1999** [comment]
Could you try using the versions specified in the README: Python 3.7, PyTorch 1.12.1, PyTorch Geometric 2.3.1, and DGL 1.0.2? Since the “Questions” dataset is an imbalanced binary classification problem, the discrepancy you observed is quite unexpected.
